src/d3a/d3a_core/util.py

```diff
@@ -486,3 +487,28 @@ def is_time_slot_in_past_markets(time_slot: DateTime, current_time_slot: DateTim
             hours=ConstSettings.SettlementMarketSettings.MAX_AGE_SETTLEMENT_MARKET_HOURS))
     else:
         return time_slot < current_time_slot
+
+
+class FutureMarketCounter:
+    """Hold a time counter for the future market.
+
+    In the future market, we only want to clear in a predefined interval.
+    """
+    def __init__(self):
+        self.last_time_dispatched = pendulum.now()
```

Comment:
I think that this will not work out as expected. It might work out in the case of Canary Networks, where the simulation time is following the real clock time, however it will not work for simulation and collaborations that are running in accelerated mode. In order to achieve that you need to pass the simulation clock time. Easiest way to get it is via the market.now member, which contains the simulation datetime.